test/intl402/RelativeTimeFormat/constructor/constructor/locales-invalid.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.RelativeTimeFormat
+description: Checks error cases for the locales argument to the RelativeTimeFormat constructor.
+info: |
+    InitializeRelativeTimeFormat (relativeTimeFormat, locales, options)
+    3. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+includes: [testIntl.js]
+features: [Intl.RelativeTimeFormat]
+---*/
+
+for (const [locales, expectedError] of getInvalidLocaleArguments()) {
+    assert.throws(expectedError, function() { new Intl.RelativeTimeFormat(locales) })
```

Comment:
I wonder if `getInvalidLocaleArguments` could return a third item to identify as the message. `assert.throws` accepts a third arg for the message. 

In the case of using loops, any failure could be way easier to be found given a description.

Just providing the `locales` won't work as this would throw in the given objects with a symbol inside.

Probably too much of a burden.